site/src/docs-components/color/ZenGrays.tsx

```diff
@@ -1,20 +1,14 @@
 import * as React from "react"
```

Comment:
![Screen Shot 2021-07-16 at 11 56 55 am](https://user-images.githubusercontent.com/1621182/125880102-d8c650ca-0aa6-4fed-ac5e-ab3da37e034d.png)

There's a Gatsby error in this area of code (likely from another file)

Reply:
wait is this only when you build it locally?...CI seems fine?